devel/Github-Desktop/Portfile

```
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                Github-Desktop
```

Capitalize GitHub correctly.
```suggestion
name                GitHub-Desktop
```

---

The name of the directory that the Portfile is in should be updated to match this capitalization.